api/v0/common.go

```diff
@@ -0,0 +1,74 @@
+package v0
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+type Error struct {
+	Message string
```

Comment:
Nonblocker: I would argue having a dedicated code field here would be a good investment, since `MakeError` already takes `status` int.

Reply:
@gammazero and I had this same discussion yesterday when you left from the call. At this point we don't use codes for anything. We most probably add it in the future.